lib/tasks/verify-package-json.js

```diff
@@ -95,6 +95,10 @@ async function packageJson(config) {
 	if (exists) {
 		const file = await readFile(packageJsonPath, 'utf8');
 		const packageJson = JSON.parse(file);
+
+		if (packageJson.type !== 'module') {
```

Comment:
Are we not dropping `module` for `component` in v2? Currently an alias.
https://origami.ft.com/spec/v1/manifest/#origamitype

Reply:
https://github.com/Financial-Times/origami-website/pull/273/files#r617396987

Reply:
> Are we not dropping `module` for `component` in v2? Currently an alias.
> [origami.ft.com/spec/v1/manifest/#origamitype](https://origami.ft.com/spec/v1/manifest/#origamitype)

This is the package.json manifest, the value of the `type` field specifies to NodeJS and build tools whether the package is written using `require` or `import/export`.

Reply:
Ahhh of course, good, well at least it got me thinking about `origamiType` :)